### PR TITLE
add new fields for Site --- "short_link" and "images"

### PR DESCRIPTION
**Pull request by Baalmart**
**_WHAT DOES THIS PR DO?_**

add new fields for Site - `short_link` and `images`

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

- Jira cards
    - [AN-257]

**_HOW DO I TEST OUT THIS PR?_**
```
cd src/device-registry
npm install
npm run stage-mac
```

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

- [x] [create site](https://airqo-engineering.postman.co/workspace/internal~7a8e9fad-2d13-4d27-af27-f1f4e9a114bf/request/12513372-430b7044-9b88-44dd-80db-0e7856c6d471)
- [x] [update site](https://airqo-engineering.postman.co/workspace/internal~7a8e9fad-2d13-4d27-af27-f1f4e9a114bf/request/12513372-f979140c-0048-4834-9e24-ba237bf2ed38)

**_SCREENSHOTS?:_** 
Screenshot 2023-01-08 at 12 26 38

[AN-257]: https://airqoteam.atlassian.net/browse/AN-257?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/device-registry/models/Site.js [hunk 1]
```diff
@@ -13,6 +13,16 @@ const siteSchema = new Schema(
       trim: true,
       required: [true, "name is required!"],
     },
+    share_links: {
+      preview: { type: String, trim: true },
+      share: { type: String, trim: true },
```
Comment (NoahNsimbe) [inline src/device-registry/models/Site.js:18]: Hi @Baalmart I think let's call this `short_link` because we have a long link and a short link

Comment (Baalmart) [reply src/device-registry/models/Site.js:18]: Alright @NoahNsimbe , let me make the update asap...

Comment (Baalmart) [reply src/device-registry/models/Site.js:18]: DONE. @NoahNsimbe

**Review by 123MwanjeMike — APPROVED** [review]
LGTM

File: src/device-registry/models/Site.js (hunk 1 above)
Comment (NoahNsimbe) [reply src/device-registry/models/Site.js:18]: oh... sorry I meant changing `share` to `short_link` i.e
```
share_links: {
      preview: { type: String, trim: true },
      short_link: { type: String, trim: true },
}
```
@Baalmart

Comment (Baalmart) [reply src/device-registry/models/Site.js]: Hi @NoahNsimbe , please check out the recently pushed changes.
Screenshot 2023-01-08 at 12 26 38

File: src/device-registry/models/Site.js [hunk 2]
```diff
@@ -351,6 +361,8 @@ siteSchema.methods = {
       geometry: this.geometry,
       village: this.village,
       site_codes: this.site_codes,
+      images: this.images,
+      short_link: this.short_link,
```
Comment (NoahNsimbe) [inline src/device-registry/models/Site.js:365]: could this be a typo @Baalmart

File: src/device-registry/models/Site.js [hunk 3]
```diff
@@ -485,6 +497,8 @@ siteSchema.statics = {
           landform_90: 1,
           aspect: 1,
           status: 1,
+          images: 1,
+          short_link: 1,
```
[the remark this answers is not in the document]
Comment (NoahNsimbe) [reply src/device-registry/models/Site.js:501]: could this be a typo @Baalmart

[the remark this answers is not in the document]
Comment (Baalmart) [reply src/device-registry/models/Site.js]: Thanks @NoahNsimbe it has been addressed!

**Review by NoahNsimbe — APPROVED** [review]
LGTM

**Pull request merged**